### DISCUSSION TITLE
Buyer persona - Web

### DISCUSSION THREAD

**Discussion by LauraHenao27** [Launch-X-Latam/MisionFrontEnd#148] [Ideas]
Hola todos y todas, un abrazo
Estuve trabajandole a los requerimientos y entregables y encontre esta maravilla para crear Buyer persona
**https://www.hubspot.es/make-my-persona**
Es igual al ejemplo que tenemos en el git, espero les sirva.

Lau

**Comment by iam-leo** [comment]
Está genial Laura, no conocía esa herramienta. Muchas gracias por compartirlo con la comunidad, nos ahorrará mucho tiempo!

**Comment by lopez-juandavid** [comment]
Muchas gracias Explorer por compartir la herramienta.

**Comment by DaveOval** [comment]
Yo intente usarlo pero no me gusto el resultado y termine haciéndolo en figma, saludos.

**Reply by AndrjsM** [reply]
Muchas gracias @LauraHenao27, esta genial

**Comment by CristopherSandoval** [comment]
Gracias!!! ya tenía una hecha pero con esta la  hago mejor! :D

**Comment by luisalbertmschz** [comment]
> Hola todos y todas, un abrazo
> Estuve trabajandole a los requerimientos y entregables y encontre esta maravilla para crear Buyer persona
> **https://www.hubspot.es/make-my-persona**
> Es igual al ejemplo que tenemos en el git, espero les sirva.
> 
> Lau

Super excelente, muchas gracias por compartirla
